rest-api/etl/ddl.sql

```diff
@@ -430,6 +430,11 @@ CREATE TABLE observation
     PRIMARY KEY (observation_id)
 );
 
+CREATE INDEX ux_observation ON cdm.observation (person_id, observation_concept_id, observation_date,
```

Comment:
Are more than the first 2 cols important to index? (I know this is just moving code from another file, but it's causing me to read closer ;-))

Reply:
TBH I haven't looked. It does seem kinda unnecessary index *all* of these. :)

I'll ask Anna.